### ISSUE TITLE
Export html under Windows is currently broken

### ISSUE THREAD

**Issue by scls19fr** [marimo-team/marimo#1249]
### Describe the bug

Following https://github.com/marimo-team/marimo/issues/896#issuecomment-2079993802

export to html seems to be broken in my install.

```
> marimo export html am_modulation.py -o am_modulation.html
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "C:\Users\scelles\anaconda3\Scripts\marimo.exe\__main__.py", line 7, in <module>
  File "C:\Users\scelles\anaconda3\Lib\site-packages\click\core.py", line 1157, in __call__
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\scelles\anaconda3\Lib\site-packages\click\core.py", line 1078, in main
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "C:\Users\scelles\anaconda3\Lib\site-packages\click\core.py", line 1688, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\scelles\anaconda3\Lib\site-packages\click\core.py", line 1688, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\scelles\anaconda3\Lib\site-packages\click\core.py", line 1434, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\scelles\anaconda3\Lib\site-packages\click\core.py", line 783, in invoke
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\scelles\anaconda3\Lib\site-packages\marimo\_cli\export\commands.py", line 97, in html
    (html, _filename) = asyncio.run(
                        ^^^^^^^^^^^^
  File "C:\Users\scelles\anaconda3\Lib\asyncio\runners.py", line 190, in run
    return runner.run(main)
           ^^^^^^^^^^^^^^^^
  File "C:\Users\scelles\anaconda3\Lib\asyncio\runners.py", line 118, in run
    return self._loop.run_until_complete(task)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\scelles\anaconda3\Lib\asyncio\base_events.py", line 653, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "C:\Users\scelles\anaconda3\Lib\site-packages\marimo\_server\export\utils.py", line 43, in run_app_then_export_as_html
    session = await run_app_until_completion(file_manager, cli_args)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\scelles\anaconda3\Lib\site-packages\marimo\_server\export\utils.py", line 92, in run_app_until_completion
    session = Session.create(
              ^^^^^^^^^^^^^^^
  File "C:\Users\scelles\anaconda3\Lib\site-packages\marimo\_server\sessions.py", line 290, in create
    return cls(
           ^^^^
  File "C:\Users\scelles\anaconda3\Lib\site-packages\marimo\_server\sessions.py", line 330, in __init__
    self.message_distributor.start()
  File "C:\Users\scelles\anaconda3\Lib\site-packages\marimo\_utils\distributor.py", line 54, in start
    asyncio.get_event_loop().add_reader(
  File "C:\Users\scelles\anaconda3\Lib\asyncio\events.py", line 530, in add_reader
    raise NotImplementedError
NotImplementedError
(base) C:\.....\marimo [main ≡ +1 ~0 -0 !]> marimo --version
0.4.5
```

### Environment

```
marimo env
{
  "marimo": "0.4.5",
  "OS": "Windows",
  "OS Version": "10",
  "Processor": "Intel64 Family 6 Model 142 Stepping 12, GenuineIntel",
  "Python Version": "3.11.5",
  "Binaries": {
    "Browser": "124.0.6367.91",
    "Node": "v22.0.0"
  },
  "Requirements": {
    "click": "8.1.7",
    "importlib-resources": "missing",
    "jedi": "0.18.1",
    "markdown": "3.6",
    "pymdown-extensions": "10.8",
    "pygments": "2.15.1",
    "tomlkit": "0.12.4",
    "uvicorn": "0.29.0",
    "starlette": "0.37.2",
    "websocket": "missing",
    "typing-extensions": "4.7.1",
    "black": "0.0"
  }
}
```

**Comment by akshayka** [comment]
@scls19fr , thank you for reporting this. I'll make a fix and improve our tests.

**Comment by akshayka** [comment]
I've fixed this in #1252. I will cut a new release soon and write back once it's out.

**Comment by akshayka** [comment]
@scls19fr, I've just released version 0.4.7. It should be on PyPI momentarily. I'll close this issue for now, but if you still run into problems, please do re-open it.

Thanks again for the bug report!

**Comment by scls19fr** [comment]
LGTM
